backend/app/views/spree/admin/tax_categories/index.html.erb

```diff
@@ -34,18 +33,47 @@
     </tr>
   </thead>
   <tbody>
-    <% @tax_categories.each do |tax_category|
-         @edit_url = edit_admin_tax_category_path(tax_category)
-         @delete_url = admin_tax_category_path(tax_category)
-    %>
+    <% @tax_categories.each do |tax_category| %>
       <tr id="<%= spree_dom_id tax_category %>" data-hook="tax_row" class="<%= cycle('odd', 'even')%>">
-        <td class="align-center"><%= tax_category.name %></td>
-        <td class="align-center"><%= tax_category.tax_code %></td>
-        <td class="align-center"><%= tax_category.description %></td>
-        <td class="align-center"><%= tax_category.is_default? ? Spree.t(:say_yes) : Spree.t(:say_no) %></td>
+        <td class="align-center">
```

Comment:
I think it could reduce some markup but it would drastically increase the amount of JS, no? If that's the direction we'd like to go that's fine but I think the net footprint will be much larger (since each page will require its own JS).